frontend/benefit/handler/public/locales/sv/common.json

```diff
@@ -14,7 +14,10 @@
     },
     "languageMenuButtonAriaLabel": "Ändra språk",
     "logoutLabel": "Logga ut",
-    "loginLabel": "Logga in"
+    "loginLabel": "Logga in",
+    "userAriaLabelPrefix": "Käyttäjä:",
+    "linkSkipToContent": "Hyppää pääsisältöön",
+    "menuToggleAriaLabel": "Valikko"
```

Comment:
Code style thing again :D If I correctly interpret our `.prettierrc`, we should always have trailing commas?

Reply:
This is JSON, not JS. So no, it must be valid JSON.

Reply:
Locales other than fi is not used in handler at all. The test scaffolding however needs the locales to be 1:1 with the keys. So I've just filled in with placeholders wherever necessary.

Reply:
Ah, sorry, my bad.